source/wp-content/themes/wporg-showcase-2022/patterns/_site-hero.php

```diff
@@ -0,0 +1,55 @@
+<?php
+/**
+ * Title: Site Hero (Featured)
+ * Slug: wporg-showcase-2022/site-hero
+ * Inserter: no
+ *
+ * This renders a random single sticky (hero featured) site.
+ */
+
+?>
+<!-- wp:query {"queryId":0,"query":{"perPage":"1","pages":0,"offset":0,"postType":"post","order":"desc","orderBy":"rand","author":"","search":"","exclude":[],"sticky":"only","inherit":false},"className":"is-section-site-hero"} -->
+<div class="wp-block-query is-section-site-hero">
+	<!-- wp:post-template -->
+		<!-- wp:columns {"style":{"spacing":{"blockGap":{"top":"var:preset|spacing|30","left":"var:preset|spacing|50"}}}} -->
+		<div class="wp-block-columns">
+			<!-- wp:column {"width":"24%"} -->
+			<div class="wp-block-column" style="flex-basis:24%">
+				<!-- wp:group {"layout":{"type":"constrained"}} -->
+				<div class="wp-block-group">
+					<!-- wp:spacer {"height":"var:preset|spacing|50"} -->
+					<div style="height:var(--wp--preset--spacing--50)" aria-hidden="true" class="wp-block-spacer"></div>
+					<!-- /wp:spacer -->
+
+					<!-- wp:heading {"fontSize":"heading-2"} -->
+					<h1 class="wp-block-heading has-heading-2-font-size"><?php esc_attr_e( 'Showcase', 'wporg' ); ?></h1>
+					<!-- /wp:heading -->
+
+					<!-- wp:paragraph {style":{"spacing":{"margin":{"top":"var:preset|spacing|10"}}}} -->
+					<p style="margin-top:var(--wp--preset--spacing--10)"><?php esc_attr_e( 'Star-studded sites built with WordPress', 'wporg' ); ?></p>
```

Comment:
These translations aren't attributes, is `esc_attr_e` appropriate?

Reply:
Nope, I must've copied this and forgot to change it. thanks!